src/azure-cli/setup.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+from __future__ import print_function
+from codecs import open
+from setuptools import setup
+
+VERSION = "0.0.1"
+
+# If we have source, validate that our version numbers match
+# This should prevent uploading releases with mismatched versions.
+try:
+    with open('azure/cli/__init__.py', 'r', encoding='utf-8') as f:
+        content = f.read()
+except OSError:
+    pass
+else:
+    import re, sys
+    m = re.search(r'__version__\s*=\s*[\'"](.+?)[\'"]', content)
+    if not m:
+        print('Could not find __version__ in azure/cli/__init__.py')
+        sys.exit(1)
+    if m.group(1) != VERSION:
+        print('Expected __version__ = "{}"; found "{}"'.format(VERSION, m.group(1)))
+        sys.exit(1)
+
+CLASSIFIERS = [
+    'Development Status :: 3 - Alpha',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'License :: OSI Approved :: MIT License',
+]
+
+DEPENDENCIES = [
+    'azure-cli-core',
+    'azure-cli-component',
+    'azure-cli-configure',
+    'azure-cli-feedback',
+    'azure-cli-network',
+    'azure-cli-profile',
+    'azure-cli-resource',
+    'azure-cli-role',
+    'azure-cli-storage',
+    'azure-cli-vm',
```

Comment:
Just a quick look, we don't have webapp. If it is intentionally, I will add later myself when on-boarding new commands